plugins/argocd-dashboard/templates/wave-1/argocd-terra-dashboard.ingressroute.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if eq .Values._juno_gateway_api true }}
+# The plain gateway API doesn't play nice with insecure-transport, so we are ok with "vendor lockin" here.
+# It is fairly trivial/low-cost to migrate away from this if we have to
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: "argocd-terra-dashboard"
+  namespace: "argocd"
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - match: "Host(`{{ .Values.host }}`) && PathPrefix(`{{ .Values.prefix }}`)"
+      kind: Rule
+      middlewares:
+        - name: "genesis-auth-service"
+      services:
+        # argo listens for https on 80 too
+        - name: argocd-server
+          port: 80
+          scheme: https
+          serversTransport: argocd-server-insecure-transport
+
+{{- end }}
```

Comment:
I think this is left over from gateway API? why are we updating argocd-dashboard?